tests/codemods/test_fix_task_instantiation.py

```diff
@@ -0,0 +1,195 @@
+import pytest
+from core_codemods.fix_task_instantiation import FixTaskInstantiation
+from tests.codemods.base_codemod_test import BaseCodemodTest
+
+
+class TestFixTaskInstantiation(BaseCodemodTest):
+    codemod = FixTaskInstantiation
+
+    @pytest.mark.parametrize(
+        "input_code,expected_output",
+        [
+            (
+                """
+                import asyncio
+                asyncio.Task(coro(1, 2))
+                """,
+                """
+                import asyncio
+                asyncio.create_task(coro(1, 2))
+                """,
+            ),
+            (
+                """
+                import asyncio
+                async def coro(*args):
+                    print(args)
+
+                my_coro = coro(1, 2)
+                asyncio.Task(my_coro)
+                """,
+                """
+                import asyncio
+                async def coro(*args):
+                    print(args)
+
+                my_coro = coro(1, 2)
+                asyncio.create_task(my_coro)
+                """,
+            ),
+            (
+                """
+                import asyncio
+                my_loop = asyncio.get_event_loop()
+                asyncio.Task(coro(1, 2), loop=my_loop)
+                """,
+                """
+                import asyncio
+                my_loop = asyncio.get_event_loop()
+                my_loop.create_task(coro(1, 2))
+                """,
+            ),
```

Comment:
What is the significance of referencing the loop for this codemod in this test? To ensure an event loop is running? Asking since this test case uses `asyncio.get_event_loop()` and there is a slight difference with create_task in that it calls `asyncio.get_running_loop()` rather than `asyncio.get_event_loop()`.  AFAIK `asynio.get_event_loop()` will create a loop if none is running and `asyncio.get_running_loop()` will raise an exception if no loop is running.  I'm not entirely sure it significant here, but thought I should mention it since it is a little sneaky.  For reference:

https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task
https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_running_loop

Reply:
Very sneaky indeed! 
Using `get_event_loop` in these tests is actually only for demonstration purposes. For these tests, we don't care if the loop came from `get_event_loop` or just some loop that was passed in from some other random function. All we care about is that if a loop is passed in as a kwarg to `asyncio.Task`, then we use that very loop and call `create_task` on it, and not call `asyncio.create_task`.

So your explanation actually reinforces this unit test, since we wouldn't not want to call `get_running_loop` and raise an error.

LMK if that didn't make sense or if you feel a new unit test is needed.